tests/test_dictField.py

```diff
@@ -64,6 +66,16 @@ class BadItemtype(pexConfig.Config):
         else:
             raise SyntaxError("Unsupported itemtype DictFields should not be allowed")
 
+        try:
+
+            class BadKeyCheck(pexConfig.Config):
+                d = pexConfig.DictField("...", keytype=int, itemtype=int, keyCheck=4)
+
+        except Exception:
```

Comment:
Not `ValueError`, or a `with self.assertRaises(ValueError, ...` like below?

Reply:
I agree, and that does seem more suitable to me as well. However, there are four more `try...except` blocks in this test case, both above and below, and I didn’t want to break consistency. I think we should modernize all these old tests in a new ticket.